packages/docusaurus-types/src/index.d.ts

```diff
@@ -290,6 +290,10 @@ export interface Plugin<Content = unknown> {
 export type InitializedPlugin<Content = unknown> = Plugin<Content> & {
   readonly options: Required<PluginOptions>;
   readonly version: DocusaurusPluginVersionInformation;
+  /**
+   * The absolute path to the folder containing the entry point file.
+   */
+  readonly path: string;
```

Comment:
maybe having a more explicit name would be better than a comment?

Reply:
I think it's explicit enough. The only reason I added the comment is because I would add a comment to all path-related variables in the future. Paths are always ambiguous and it's not really worth making the name super long.